src/web/src/pages/CourseExplorer.vue

```diff
@@ -55,12 +103,37 @@ export default {
     return {
       //an object with keys being the dept, and values a list of courses
       deptClassDict: {},
+      schoolsMajorDict: {},
       ready: false,
+
+      /*
+       *Used to define the order that the schools are placed into the two main
+       *columns. This could be implemented generically, but making the columns
+       *more or less "even" is NP-Hard so its more convenient to just define
+       *manually
+       */
+      schoolOrder: [
```

Comment:
what if you sort the schools by the number of courses in each school?

Reply:
That doesn't work just because the schools are pulled out in order, so if you sort by number of courses the first column would always have the two biggest schools. If we did it like that then HASS and Science would be in the first column which would be way bigger than the second.

Reply:
Makes sense carry on